### PR TITLE
Small fixes and corrections

### PR DESCRIPTION
**Pull request by Heegu-sama**
Expanded the description of attacking walls, fixed and made consistent many descriptions in the expansion and location sections by including svg images and Capitalizing certain terms.

### CODE REVIEW

**Comment by qwrtln** [comment]
Hold on, some of those were actually DEcapitalised by @RickardNi yesterday?

**Comment by Heegu-sama** [comment]
> Hold on, some of those were actually DEcapitalised by @RickardNi yesterday?

Highly possible. I'll wait for his reply.

File: sections/combat.tex [hunk 1]
```diff
@@ -95,7 +95,8 @@ \subsection*{\hypertarget{Walls}{Defending a Town With a Citadel}}
 When a Town with a Citadel is attacked, the defender adds the Wall and Gate Obstacles in any order to the Middle Row of the Combat Board after placing their Units.
 The Gate Card is \textbf{not an Obstacle to the defending player}.
 The Wall and Gate cards can be destroyed by any adjacent \includesvg[height=10px]{\svgs/unit_ground.svg} or \includesvg[height=10px]{\svgs/unit_flying.svg} Unit's attack.
-Such attacks are always successful, do not roll the Combat Die when performing them.\par
+\textbf{Note:} Such attacks are always successful and they do not follow the normal framework for attacking other units.
```
Comment (RickardNi) [inline sections/combat.tex:98]: This should be a capitalized Unit.

**Comment by RickardNi** [comment]
It looks fine from here, I don't think anything was decapitalized unless I'm missing something. Can you give example?

**Comment by qwrtln** [comment]
Nope, lines I looked at had other changes. Example: all_map_locations.tex:35 Gold->gold, towns->Towns. All good.

**Review by qwrtln — APPROVED** [review]

**Pull request merged**